setup.cfg

```diff
@@ -215,12 +215,15 @@ cylc.main_loop =
 cylc.pre_configure =
 cylc.post_install =
     log_vc_info = cylc.flow.install_plugins.log_vc_info:main
-# NOTE: Built-in xtriggers
+# NOTE: Built-in xtrigger modules
+#   - must contain a function (the xtrigger) with the same name as the module
+#   - and may contain a "validate" function to check arguments
 cylc.xtriggers =
-    echo = cylc.flow.xtriggers.echo:echo
-    wall_clock = cylc.flow.xtriggers.wall_clock:wall_clock
-    workflow_state = cylc.flow.xtriggers.workflow_state:workflow_state
-    xrandom = cylc.flow.xtriggers.xrandom:xrandom
+    echo = cylc.flow.xtriggers.echo
+    wall_clock = cylc.flow.xtriggers.wall_clock
+    workflow_state = cylc.flow.xtriggers.workflow_state
+    xrandom = cylc.flow.xtriggers.xrandom
```

Comment:
Probably worth making @ColemanTom aware of this change as they implemented this entry point in #5831 (still unreleased)

Reply:
Ping @ColemanTom - are you cool with this?

Reply:
Whatever you like/think will make things better. It looks like a simple thing to update for my pending branch at work (which we would need to test anyway).